pkg/agent/cniserver/interface_configuration_linux.go

```diff
@@ -260,13 +259,6 @@ func (ic *ifConfigurator) configureContainerLinkVeth(
 		}
 		containerIface.Mac = containerVeth.HardwareAddr.String()
 		hostIface.Mac = hostVeth.HardwareAddr.String()
-		// OVS netdev datapath doesn't support TX checksum offloading, i.e. if packet
-		// arrives with bad/no checksum it will be sent to the output port with same bad/no checksum.
-		if ic.ovsDatapathType == ovsconfig.OVSDatapathNetdev {
```

Comment:
@antoninbas the code is added for ovs offload, isn't it needed any more?

Reply:
OVS offload? I think the only OVS offload we support today relies on the kernel datapath and should not be using this code. If you are talking about other types of offload, such as with DPDK, we don't support that at the moment, and we decided for #3677 that we would add this code back if we ever need it.

But we should make sure that the `jenkins-hw-offload` job succeeds before merging this PR.

Reply:
I just checked the Git history, and git blame shows that this code was "added" for OVS offload, but IIRC the code was already there and was moved as part of some code refactoring when adding OVS offload support.